autopush_rs/src/util/megaphone.rs

```diff
@@ -114,11 +122,27 @@ impl ServiceChangeTracker {
         svc_change_tracker
     }
 
+    /// Creates a new `ServiceChangeTracker` initialized from a Megaphone API server version set
+    /// as provided as the fetch URL.
+    ///
+    /// This method uses a synchronous HTTP call.
+    pub fn with_api_services(url: &str) -> reqwest::Result<ServiceChangeTracker> {
+        let client = reqwest::Client::builder()
```

Comment:
I wonder if perhaps the `Client` may be best stored in the ambient server configuration rather than created on each iteration?

Reply:
This one is a one-off startup request, so I didn't store the client. The recurring updater does store the client between requests though.